packages/wallet/src/redux/sagas/channel-updater.ts

```diff
@@ -0,0 +1,20 @@
+import * as selectors from '../selectors';
+import { select, call, put } from 'redux-saga/effects';
+import {
+  getAdjudicatorHoldings,
+  getProvider,
+  getAdjudicatorOutcome,
+} from '../../utils/contract-utils';
+import { channelUpdate } from '../actions';
+import { BigNumber, bigNumberify } from 'ethers/utils';
+
+export function* channelUpdater() {
```

Comment:
This saga is responsible for updating the `adjudicatorState` for channels after they're loaded fro storage. Since we only store the `channelStore` in localStorage we need to update the `adjudicatorStore` with information on the funding level/ whether a channel is finalized from the blockchain.